karma.conf.js

```diff
@@ -72,7 +73,7 @@ module.exports = function (config) {
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
         // you can also add Chrome or other browsers too
-        browsers: ['PhantomJS'],
+        browsers: ['Chrome'],
```

Comment:
No, because I made the test task explicitly pass 'PhantomJS'. I thought that Chrome would be a better default in general, but maybe that's not right.